Sources/App/Controllers/PackageController+routes.swift

```diff
@@ -59,14 +59,15 @@ struct PackageController {
         case css
         case data
         case documentation
+        case images
         case js
         case themeSettings
 
         var contentType: String {
             switch self {
                 case .css:
                     return "text/css"
-                case .data, .themeSettings:
+                case .images, .data, .themeSettings:
                     return "application/octet-stream"
```

Comment:
I'm not sure if there's a downside to return such a broad content-type for images but it seems to work ok. It would certainly be awkward to have to fiddle with explicit png, jpg etc content types based on the path.

Reply:
Let's do this until it breaks!